### DISCUSSION TITLE
Please add a way to stop the execution on Android when it gives an error

### DISCUSSION THREAD

**Discussion by Infrared100** [godotengine/godot-proposals#8981] [Platforms]
When I execute the game I can press back on Android to go back to the editor, but when it gives me an error the back button doesn't go back to the editor anymore, I have to close and restart Godot, it's hard to find what causes an error on Android because of this, I have to guess until I find what was.
Some ideas:
- a push notification to go back to the editor
- automatically go back to the editor when an error is found 
- maybe add as an debbuger option 
- some other way

Thank you very much for your help.

English isn't my first language. I do not have a computer, I'm using Godot on Android only. I'm using a floating transparent keyboard and changed some of the default settings (e.g. disabled the text editor minimap to remove clutter\have more space), I've found Godot for Android surprisingly usable other than the debbuger, thank you.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot/issues/77966, as this is a bug.
